client/my-sites/plan-features/style.scss

```diff
@@ -158,6 +158,7 @@ $plan-features-sidebar-width: 272px;
 
 .plan-features__table-item.is-bottom-buttons {
 	padding-top: 33px;
+	vertical-align: bottom;
```

Comment:
I think `vertical-align: top` would be preferable. I'd like to see the action buttons all on one line, like this:
![action buttons](https://cloudup.com/cuNzdzyR9G7+)